l3build-check.lua

```diff
@@ -936,11 +944,13 @@ function check(names)
     end
     -- Actually run the tests
     print("Running checks on")
+    local failurelist = {}
```

Comment:
failure_list

Reply:
The function almost exclusively uses names without underscores.